custom_components/flashforge_adventurer_3/sensor.py

```diff
@@ -152,3 +169,32 @@ def icon(self) -> str:
     @property
     def unit_of_measurement(self) -> str:
         return '%'
+
+class FlashForgeAdventurer5TemperatureSensor(BaseFlashForgeAdventurer3Sensor):
```

Comment:
Nozzle and bed temperatures are kept as attributes on the state sensor, and can be accessed via it. I don't think this new sensor is necessary to be part of the integration.

Reply:
I thought it was somewhat odd in my testing that the nozzle temperature was the only one with its own sensor. 🤔

Reply:
Got it. I'll remove it. :)